src/scss/includes/components/panel.scss

```diff
@@ -68,7 +73,7 @@ $bottom-margin: 40;
     width: 100vw;
     height: 50%;
     position: absolute;
-    background: #f4f6fa;
+    background: #ffffff;
```

Comment:
White or not is currently controlled by a prop which sets the `--white` variant on `.panel` (see some selectors below). So if all panels are white now, this means we should also remove all the code associated to that.

The best way here is just to set the `white` prop on `<CategoryPanel>`, and we do the cleaning in another PR. As to clean things up entirely we would have many files not related to CategoryPanel to change, and this would make this PR more confusing.

Reply:
The variable is rather `$background` right ?